src/Components/App.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/jsx-wrap-multilines */
+/* eslint-disable quotes */
+/* eslint-disable import/extensions */
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Categories from "./Categories";
+import Header from "./Header";
+import AddBook from "./AddBook";
+import DisplayBooks from "./DisplayBooks";
+
+const App = () => (
+  <Routes>
+    <Route
+      path="/"
+      element={
+        <>
+          <Header />
+          <DisplayBooks />
+          <AddBook />
+        </>
+      }
+    />
+    <Route path="/categories" element={<Categories />} />
+  </Routes>
+);
+
+export default App;
```

Comment:
- [ ] Kindly remove the linter disabling here and fix the resulting errors. It is often not a good idea to disable linters as they help write standardized code.

- [ ] As suggested earlier kindly remove the AddBook component here. It should be in the DisplayBooks component.

- [ ] We can also re-organize the Header placement to make it available throughout the app.

  Suggested code.
  ```
  import React from 'react';
  import { Route, Routes } from 'react-router-dom';
  import Categories from './Categories';
  import Header from './Header';
  import DisplayBooks from './DisplayBooks';
  
  const App = () => (
    <>
      <Header />
      <Routes>
        <Route path="/" element={<DisplayBooks />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </>
  );
  
  export default App;
  ```